docroot/modules/custom/webform_custom_submissions/src/TravelformHandler.php

```diff
@@ -0,0 +1,486 @@
+<?php
+
+namespace Drupal\webform_custom_submissions;
+
+use GuzzleHttp\Client;
+use GuzzleHttp\Psr7\Response;
+use \Exception;
+
+require_once('../travel_services_config.inc');
+
+class TravelformHandler {
+
+  protected $create_issue_url;
+  protected $domestic_project;
+  protected $international_project;
+  protected $submission_client;
+  protected $username;
+  protected $vouchers_project;
+  protected $field_helper;
+
+  public function __construct() {
+    global $CREATE_ISSUE_URL;
```

Comment:
These will not be GLOBALS, but instead injected configurations. Not essential now though.